src/styles/calendar.scss

```diff
@@ -0,0 +1,34 @@
+.calendar {
+  margin: 100px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $calendarGap;
+  padding: $calendarPadding;
+  width: $calendarMaxWidth;
+
+  &__day {
+    height: $cellHeight;
+    box-sizing: border-box;
+    width: $cellHeight;
+    position: relative;
+    border: 1px solid black;
+    background-color: #eee;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: 0.5s;

```

Comment:
don't use it like that, because it's the same as transition-duration: 0.5s or transition: all 0.5s
It's too hard for the browser
just do like transition: transform $transitionDuraiton

Also, Check this
![image](https://user-images.githubusercontent.com/91411692/223088141-c2324689-93ea-4a94-855e-cb29ca349c7b.png)
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Also, create variable for $transitionDuraiton
```suggestion
    background-color: #eee;
  
    transition: transform $transitionDuraiton;
    
    &:hover {
      background-color: pink;
      transform: translateY(-20px);
```